charts/pipelines-library/values.yaml

```diff
@@ -324,6 +324,11 @@ dashboard:
       # -- Uncomment it to enable tekton-dashboard OIDC on EKS cluster
       # nginx.ingress.kubernetes.io/auth-signin: 'https://<oauth-ingress-host>/oauth2/start?rd=https://$host$request_uri'
       # nginx.ingress.kubernetes.io/auth-url: 'http://oauth2-proxy.<edp-project>.svc.cluster.local:8080/oauth2/auth'
+    # -- Defines the base URL for the portal.
+    # -- If not specified the pattern "portal-[namespace].[global DNS wildcard]" will create a URL like "portal-namespace.example.com"
```

Comment:
If not specified the pattern "tekton-[namespace].[global DNS wildcard]" will create a URL like "tekton-namespace.example.com"